packages/artillery/package.json

```diff
@@ -83,20 +83,20 @@
     "email": "npm@veldstra.org"
   },
   "dependencies": {
-    "@artilleryio/int-commons": "latest",
-    "@artilleryio/int-core": "latest",
+    "@artilleryio/int-commons": "*",
+    "@artilleryio/int-core": "*",
     "@artilleryio/platform-fargate": "^2.0.0",
     "@aws-sdk/credential-providers": "^3.127.0",
     "@oclif/core": "^2.8.11",
     "@oclif/plugin-help": "^5.2.11",
     "@oclif/plugin-not-found": "^2.3.1",
     "archiver": "^5.3.1",
-    "artillery-engine-playwright": "^0.3.0",
```

Comment:
@hassy one question I had is if there was any reason this engine in particular used `^0.3.0` instead of `latest`, unlike the others? I put it as `*` like the others.

Reply:
iirc it's to guard against potentially backwards incompatible changes in Playwright but I'm not sure how valid that concern is